ci-build/docker-build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+shopt -s extglob
+
+TMP_DIR=$(mktemp -d)
+
+function main {
+  local here
+  here=$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )
+
+  cp "$here/Dockerfile" "$TMP_DIR"
+  cd "$here/.."
+  cp -r !(.*|html|output|Dockerfile) "$TMP_DIR"
```

Comment:
This is a bit convoluted, can you add a comment saying what things need to be copied into TMP_DIR?

Reply:
Done, PTAL to see if it's helpful